ntoskrnl/io/pnpmgr/arbs.c

```diff
@@ -92,17 +343,67 @@ IopMemInitialize(VOID)
 
     DPRINT("IopRootMemArbiter %p\n", &IopRootMemArbiter);
 
+    IopRootMemArbiter.UnpackRequirement = IopGenericUnpackRequirement;
+    IopRootMemArbiter.PackResource = IopGenericPackResource;
+    IopRootMemArbiter.UnpackResource = IopGenericUnpackResource;
+    IopRootMemArbiter.ScoreRequirement = IopGenericScoreRequirement;
+
+    IopRootMemArbiter.FindSuitableRange = IopMemFindSuitableRange;
```

Comment:
Why arbiters have different set of functions assigned?

Reply:
For 'MEM' and 'PORT' arbiters, some functions are common (they have the same functionality).